hw/ip/kmac/lint/sha3.vlt

```diff
@@ -0,0 +1,10 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+//
+// waiver file for kmac
+
+`verilator_config
+
+// index_z is of type int but we only use bits 5:0.
+lint_off -rule UNUSED -file "*/rtl/keccak_2share.sv" -match "*index_z*"
```

Comment:
This is ok, I guess, but it might be better to just make `index_z` a localparam, which doesn't need anything Verilator-specific.

Reply:
This doesn't work because `index_z` is not assigned a constant expression but a loop variable instead. Inside the loop itself the assigned value is fixed but maybe the problem is that we are also inside a function here.

Reply:
Huh, that's odd. I tried it and saw that the lint error went away, but I didn't check that the code actually worked(!). Do we just need to label the loop body?

Reply:
The lint warning went away but I got a tool error. But maybe I am just too stupid for this. If you can send me your code snippet, I'll try that out.

Reply:
Oh! No, it seems that I'm being stupid and didn't read the output properly! Yep, I get the UNSUPPORTED error too.